### DISCUSSION TITLE
how can install a community extension in flossboxin

### DISCUSSION THREAD

**Discussion by oussamaghrib** [FreshRSS/FreshRSS#5811] [Q&A] [answered]
is there a way to install community extensions (specifically greader) in a cloud provider like flossboxin?

**Comment by Alkarex** [comment, marked as answer]
Quick partial answer already: the GReader API is built-in in FreshRSS and does not require any extension.

**Comment by Alkarex** [comment]
I am not familiar with this cloud provider. How do you deploy FreshRSS there? (e.g. docker-compose, single-click...)

**Reply by oussamaghrib** [reply]
hi. thanks for the reply. these cloud providers are just like a feedly for example, but they are free (most of them at least) and they run freshrss just like you are hosting yourself

https://freshrss.org/cloud-providers.html

**Reply by Alkarex** [reply]
Ah ok. Extensions can only be installed by an administrator of an instance, not by users (otherwise that would allow running untrusted code)
